src/toolbars/MeterToolBar.cpp

```diff
@@ -72,6 +72,19 @@ MeterToolBar::~MeterToolBar()
 {
 }
 
+MeterToolBars MeterToolBar::GetToolBars(AudacityProject &project)
+{
+   return MeterToolBars{
+      Get(project, PlayMeterBarID),
+      Get(project, RecordMeterBarID)
+   };
+}
+
+const MeterToolBars MeterToolBar::GetToolBars(const AudacityProject &project)
+{
+   return GetToolBars( const_cast<AudacityProject&>( project ));
```

Comment:
This use of reference wrappers is nice.

However this function exposes a vector of non-const reference (wrappers) to toolbars and only makes the returned vector const.

Not your intention I think:  so I think you must define this return type:

```
using ConstMeterToolBars = std::vector< std::reference_wrapper<Const MeterToolBar> >;
```

... and not call through to the other function.

Reply:
However, you don't even use the const overload, I think